aten/src/ATen/cpu/vec/vec256/vec256_bfloat16.h

```diff
@@ -581,32 +581,32 @@ Vectorized<BFloat16> inline operator^(const Vectorized<BFloat16>& a, const Vecto
   return _mm256_xor_si256(a, b);
 }
 
-Vectorized<BFloat16> Vectorized<BFloat16>::eq(const Vectorized<BFloat16>& other) const {
+inline Vectorized<BFloat16> Vectorized<BFloat16>::eq(const Vectorized<BFloat16>& other) const {
```

Comment:
Please add the comment to PR description explaining why adding inline is necessary